chart/openfaas/templates/alertmanager-dep.yaml

```diff
@@ -26,8 +26,18 @@ spec:
           - "alertmanager"
           - "--config.file=/alertmanager.yml"
           - "--storage.path=/alertmanager"
+        livenessProbe:
+          httpGet:
+            path: /#/status
+            port: 9093
+          timeoutSeconds: 30
```

Comment:
Curious where 30 seconds comes from? Could AlertManager take long to return? 

The value for `periodSeconds` is `10` in the [docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)

Reply:
The value comes from the official chart https://github.com/helm/charts/tree/master/stable/prometheus also the Prom operator has the same values.